package.json

```diff
@@ -17,11 +17,11 @@
     "apache"
   ],
   "scripts": {
-    "test": "npm run eslint && npm run test-unit && npm run test-e2e",
+    "test": "npm run lint && npm run test-unit && npm run test-e2e",
     "test-unit": "jasmine",
     "test-e2e": "jasmine spec/e2e/endtoend.spec.js",
     "cover": "nyc jasmine",
-    "eslint": "eslint . \"+(bin|template)/**/!(*.*)\""
+    "lint": "eslint . \"+(bin|template)/**/!(*.*|gitignore)\""
```

Comment:
Same. Can't we use `eslint .`?

Reply:
if I remember correctly, its because some of the files are not JS files but are written with JS.. For example `check_all`, `check_req`... Written this will will force them to be checked as well.